deepchem/models/tests/test_ferminet.py

```diff
@@ -12,8 +12,13 @@
 except ModuleNotFoundError:
     pass
 
+try:
```

Comment:
You shouldn't need this check (see the try/catch above)

Reply:
done